### DISCUSSION TITLE
How can I set a custom default value other than "N/A" when I load an empty cell from a CSV (or other source)

### DISCUSSION THREAD

**Discussion by mfabiani-av** [atoti/atoti#585] [Q&A] [answered]
How can I set a custom default value other than "N/A" when I load an empty cell from a CSV (or other source)?

One suggestion I had was to load the data into a pandas df, and then load the df. But considering that in Java we can set a default value for a field in the datastore, is it possible to do the same in atoti Python?

**Comment by bloussou** [comment]
Hello,
There is currently no way to change it, loading before in pandas and then in atoti is one workaround for this.

What is the use case? Why do you want to change this default value?

**Reply by mfabiani-av** [reply]
In general I have seen many projects where the default value is not "N/A", but an actual value.
In this particular case, I was trying to set it to `_DATAMEMBER_` because my table contains the levels of an unbalanced hierarchy.
I could use pandas, just wondering if it is memory efficient to load all the data in a df and then load it into atoti, considering in the backend there is a simple way of doing this.

**Comment by bloussou** [comment, marked as answer]
Hello,
From atoti `0.6.6` you will have the possibility to choose default values for each non nullable columns :tada: like this:
```python
table = session.create_table(
    "Default values",
    types={"column": tt.type.STRING},
    default_values={"column": "MY_DEFAULT"},
)
```

**Reply by mfabiani-av** [reply]
Nice!
